c2corg_ui/externs/appx.js

```diff
@@ -275,6 +275,18 @@ appx.Image;
 
 /**
  * @typedef {{
+ *   locales: Array.<Object>,
```

Comment:
Actually all document typedef use ``locales: Array.<Object>`` (not only Image or Area).
And the definition of ``appx.DocumentLocale`` seems a bit too restrictive to be used instead.
```
/**
 * @typedef {{
 *   title: string,
 *   title_prefix: ?string,
 *   lang: string
 * }}
 */
appx.DocumentLocale;
```
https://github.com/c2corg/v6_ui/blob/master/c2corg_ui/externs/appx.js#L60-L67